src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1435,3 +1435,45 @@ function CategoricalArrays.categorical(df::AbstractDataFrame,
     end
     DataFrame(newcols, _names(df), copycols=false)
 end
+
+"""
+    flatten(df::AbstractDataFrame, veccol::Union{Integer, Symbol})
+
+When column `veccol` of `df` has iterable elements that define `length`, for example a `Vector` 
+of `Vector`s. Returns a DataFrame where each element of `veccol` is flattened. 
```

Comment:
```suggestion
of `Vector`s. Returns a `DataFrame` where each element of `veccol` is flattened. 
```

Can you also explain what "flattened" means?